docs/api/cozy-client.md

```diff
@@ -1431,6 +1434,19 @@ See https://docs.cozy.io/en/tutorials/data/queries/#sort-data-with-mango
 | --- | --- |
 | sort|selector|indexedFields | <code>object</code> | 
 
+<a name="QueryDefinition+checkSelector"></a>
+
+### queryDefinition.checkSelector(selector)
+Check the selector predicates.
+
+It is useful to warn the developer when a partial index might be used.
+
+**Kind**: instance method of [<code>QueryDefinition</code>](#QueryDefinition)  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| selector | <code>object</code> | The selector definition |
+
```

Comment:
When reading this I am wondering what is returned by checkSelector ? Is it a boolean value ? Can it be precised ?